<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Patcher</string>
  </property>
  <property name="accessibleDescription">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Ver1Btn">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>80</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать файл</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Ver1Path">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>341</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Ver1Label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите путь оригинального файла:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Ver2Label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите путь модифицированного файла:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Ver2Path">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>341</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="Ver2Btn">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>160</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать файл</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CreatePatchBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать патч</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ApplyPatchBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Применить патч</string>
    </property>
   </widget>
   <widget class="QLabel" name="PatchPathLabel">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Патч создан по пути:</string>
    </property>
   </widget>
   <widget class="QLabel" name="PatchPathPath">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>250</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="PatchingPath">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>341</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="PatchingBtn">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>380</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать файл</string>
    </property>
   </widget>
   <widget class="QLabel" name="PatchingLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите путь модифицируемого файла:</string>
    </property>
   </widget>
   <widget class="QLabel" name="PatchLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите путь патч-файла:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PatchPath">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>341</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="PatchBtn">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>460</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать файл</string>
    </property>
   </widget>
   <widget class="QLabel" name="CreatePatchLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Создание патча</string>
    </property>
   </widget>
   <widget class="QLabel" name="ApplyPatchLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Применение патча</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ExitBtn">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>510</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Выход</string>
    </property>
   </widget>
   <zorder>Ver1Btn</zorder>
   <zorder>Ver1Path</zorder>
   <zorder>Ver1Label</zorder>
   <zorder>Ver2Label</zorder>
   <zorder>Ver2Path</zorder>
   <zorder>Ver2Btn</zorder>
   <zorder>ApplyPatchBtn</zorder>
   <zorder>CreatePatchBtn</zorder>
   <zorder>PatchPathLabel</zorder>
   <zorder>PatchPathPath</zorder>
   <zorder>PatchingPath</zorder>
   <zorder>PatchingBtn</zorder>
   <zorder>PatchingLabel</zorder>
   <zorder>PatchLabel</zorder>
   <zorder>PatchPath</zorder>
   <zorder>PatchBtn</zorder>
   <zorder>CreatePatchLabel</zorder>
   <zorder>ApplyPatchLabel</zorder>
   <zorder>ExitBtn</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>512</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
